File: Selenium/Activities/Activity13.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.remote.webelement import WebElement
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=service) as driver:
    driver.get("https://v1.training-support.net/selenium/tables")
    print("Title is :", driver.title)
    rows = driver.find_elements(By.XPATH,"//table[@class ='ui celled striped table']/tbody/tr")
    print("No.of Rows is :",len(rows))
    cols = driver.find_elements(By.XPATH,"//table[@class ='ui celled striped table']/tbody/tr[1]/td")
    print("No.of columns :",len(cols))
    Third_row_col = driver.find_elements(By.XPATH,"//table[@class ='ui celled striped table']/tbody/tr[3]/td")
    for cell in Third_row_col:
        print(cell.text)
    sec_col = driver.find_element(By.XPATH,"//table[@class ='ui celled striped table']/tbody/tr[2]/td[2]")
    print(sec_col.text)

driver.close()

File: Selenium/Activities/Activity13.java
package activities;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import io.github.bonigarcia.wdm.WebDriverManager;

import java.util.List;

public class Activity13 {
    public static void main(String[] args) throws InterruptedException {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/tables");
        System.out.println("Title is :" + driver.getTitle());
        List<WebElement> rows = driver.findElements(By.xpath("//table[@class ='ui celled striped table']/tbody/tr"));
        System.out.println("No.of rows :" + rows.size());
        List<WebElement> columns = driver.findElements(By.xpath("//table[@class ='ui celled striped table']/tbody/tr[1]/td"));
        System.out.println("No.of columns :" + columns.size());
        List<WebElement> Third_row_col = driver.findElements(By.xpath("//table[@class ='ui celled striped table']/tbody/tr[3]/td"));
        for (WebElement cell : Third_row_col) {
            System.out.println(cell.getText());
        }
        WebElement sec_col =driver.findElement(By.xpath("//table[@class ='ui celled striped table']/tbody/tr[2]/td[2]"));
        System.out.println(sec_col.getText());
        driver.close();
    }
}